lib/libutee/tee_api_operations.c

```diff
@@ -279,6 +282,7 @@ TEE_Result TEE_AllocateOperation(TEE_OperationHandle *operation,
 		op->info.operationClass = TEE_OPERATION_ASYMMETRIC_SIGNATURE;
 #endif
 	op->info.mode = mode;
+	op->info.digestLength = (uint32_t) digestLength;
```

Comment:
Why not use `tee_get_digest_size()` unconditionally here instead? It will give `0` for the other algorithms any way.
By the way, the cast is not needed.

Reply:
Indeed, can actually be called unconditionally